okhttp/src/main/java/com/squareup/okhttp/Cache.java

```diff
@@ -270,6 +270,19 @@ private void abortQuietly(DiskLruCache.Editor editor) {
   }
 
   /**
+   * Initialize the cache. This will include reading the journal files from
+   * the storage and building up the necessary in-memory cache information.
+   *
+   * The initialization time may vary depending on the journal file size and
+   * the current actual cache size. The application needs to be aware of calling
+   * this function during the initialization phase and preferrably in a background
+   * worker thread.
+   */
+  public void initialize() throws IOException {
+    cache.initialize();
```

Comment:
should we add the synchronized key word to the DiskLruCache::initialize() method or surround the initialize() here with synchronized(cache) ?

Looks like the reason the DiskLruCache::initialize() does not have synchronized keyword is because it is internal to the DiskLruCache class and all the callers of this method have already had the synchronized blocks.